### DISCUSSION TITLE
Trying to deploy my React project that uses React-Router

### DISCUSSION THREAD

**Discussion by calvinalee2006** [vercel/community#2172] [Help]
### Summary

I am currently having issues deploying my project that uses React Router. I have added a vercel.js file, but I can not figure out the proper configuration to deploy my project. I have read  documentation from stackoverflow posts and other documentation, but I have had no success with deploying my application.  
I have tried other different ways of deployment such as using React HashRouter ( which is not very suggested) , removing and re-installing node modules and my package.json file, but again no luck. Would anyone have an idea of the problem? 

### Additional information

When I first began I received the message:
NPM Error Code 127

Currently, when using vercel, I get the fault code:
BUILD_UTILS_SPAWN_SIGKILL: Command "vercel build" exited with SIGKILL

I tried this package update for my code:
 npm i -g vercel vercel --prod 
 to see if it fixed the problem, but it did not work.

### Example

https://github.com/calvinalee2006/my-exercise-app

**Comment by sundaram2021** [comment]
One possible cause of this issue could be a configuration problem with your vercel.js file becuase you have received these message   "Currently, when using vercel, I get the fault code:
BUILD_UTILS_SPAWN_SIGKILL: Command "vercel build" exited with SIGKILL " .
 Make sure that you have set up your routes correctly and that you have specified the correct build command. It is also possible that you may need to configure your serverless function to work with React Router.
